src/admin-views/commerce/orders/single/order-items-metabox-item.php

```diff
@@ -17,7 +17,7 @@
 <tr class="tec-tickets-commerce-single-order--items--table--row">
 	<td><?php echo esc_html( $ticket->name ); ?></td>
 	<td class="tribe-desktop-only"><?php echo 'series_pass' === $ticket->type ? esc_html__( 'Series Pass', 'event-tickets' ) : esc_html__( 'Standard Ticket', 'event-tickets' ); ?></td>
-	<td class="tec-tickets-commerce-single-order--items--table--row--info-column"><!-- @todo dpan: this is were Refunded would go --></td>
+	<td class="tec-tickets-commerce-single-order--items--table--row--info-column"><?php /* @todo dpan: this is were Refunded would go */ ?></td>
```

Comment:
Should this be a hook or entrypoint then?

Reply:
@Camwyn I don't know, I don't have much context other than the todo line. Is `dpan` @dpanta94 or someone else?